backend/src/routes/index.js

```diff
@@ -78,7 +78,7 @@ router.get('/event-codes/availability', ensureUser, ensureLogin, rateLimiter(),
   res.send(!(await Event.exists({ code })))
 })
 
-router.get('/events', ensureUser, rateLimiter({ keys: 'user._id' }), async (req, res, next) => {
+router.get('/events', ensureUser, rateLimiter(), async (req, res, next) => {
```

Comment:
we need a new middleware here that uses `user._id` as a key if the user is logged in or the default keys if the user is unauthenticated